EIPS/eip-5089.md

```diff
@@ -0,0 +1,340 @@
+---
+eip: 5089
+title: Principal Token Standard
+description: A standard for principal tokens (zero-coupon tokens) that are redeemable for a single underlying ERC-20 token at a future timestamp.
+author: Julian Traversa (@JTraversa), Robert Robbins (@robrobbins)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-principal-token-standard/1
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-01
+requires: 20, 2612
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard API for principal tokens --
+representing ownership of an underlying [ERC-20](./eip-20.md) token at a future timestamp.
+This standard is an extension on the ERC-20 token that provides basic functionality for depositing
+and withdrawing tokens and reading balances and the ERC-2612 standard that provides
+EIP-712 signature based approvals.
```

Comment:
Some general copyediting, plus adding links to mentioned EIPs:

```suggestion
The following standard defines an API for principal tokens (also known as zero-coupon tokens) that represent ownership of an underlying [ERC-20](./eip-20.md) token at a future timestamp. This standard is an extension on the ERC-20 token that provides basic functionality for depositing/withdrawing tokens, and reading balances; and the [ERC-2612](./eip-2612.md) standard that provides [EIP-712](./eip-712.md) signature based approvals.
```

Reply:
Fixed